.github/workflows/nightly.yml

```diff
@@ -92,5 +92,5 @@ jobs:
           UNIVERSAL_DEBUG_APK: app/build/outputs/apk/debug/*universal*.apk
         run: |
           mkdir $DATE
-          cp $UNIVERSAL_DEBUG_APK $DATE
+          cp $UNIVERSAL_DEBUG_APK $DATE/kiwix-android_multiarch_$DATE.apk
```

Comment:
The above line copies APK from the path. We should change the name in build.gradle file.

Reply:
hi @gouri-panda , Done